### ISSUE TITLE
0.39.1 wrong multiline string literal indentation

### ISSUE THREAD

**Issue by antiraum** [nicklockwood/SwiftFormat#362]
Version 0.39.1 wrongly idents the following code containing multiline string literals:

```
        XCTAssertTrue(concert.shortDescriptionHTML!.hasPrefix("""
        In a way, this concert looks ahead to a new era
        """))
```

is changed to

```
        XCTAssertTrue(concert.shortDescriptionHTML!.hasPrefix("""
        In a way, this concert looks ahead to a new era
            """))
```

In version 0.39.0 this didn't happen.

Thanks a lot for swiftformat and the effort you are putting into it.

**Comment by nicklockwood** [comment]
@antiraum thanks for reporting. I'll fix it asap.

**Comment by nicklockwood** [comment]
@antiraum fixed in 0.39.2. Thanks again!
